<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="edtNumber">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1234</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Student Number:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spnNumbe">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>46</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>50</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Mark:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtCode">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>COS3711</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Module Code:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
